Blob validation fails: "Could not determine revocation status"

I am trying to experiment with `webauthn-server-attestation`, but even the "standard" way of downloading blobs fails for me, leaving me stumped.

```java
package toy;

import java.util.Optional;
import java.util.function.Consumer;
import java.util.function.Supplier;

import com.yubico.fido.metadata.FidoMetadataDownloader;
import com.yubico.webauthn.data.ByteArray;

public class Test {
  private static class MemoryCache<T> implements Supplier<Optional<T>>, Consumer<T> {
    Optional<T> v = Optional.empty();
    @Override public void accept(T t) { this.v = Optional.of(t); }
    @Override public Optional<T> get() { return this.v; }
  }

  public static void main(String[] args) {
    final MemoryCache<ByteArray> trustRootCache = new MemoryCache<>();
    final MemoryCache<ByteArray> blobCache = new MemoryCache<>();
    
    final var downloader = FidoMetadataDownloader.builder()
      .expectLegalHeader("Retrieval and use of this BLOB indicates acceptance of the appropriate agreement located at https://fidoalliance.org/metadata/metadata-legal-terms/")
      .useDefaultTrustRoot()
      .useTrustRootCache(trustRootCache, trustRootCache)
      .useDefaultBlob()
      .useBlobCache(blobCache, blobCache)
      .build();
    try {
      downloader.loadCachedBlob();
    } catch (Exception e) {
      e.printStackTrace();
    }
  }
}
```

This results in:
```
java.security.cert.CertPathValidatorException: Could not determine revocation status
        at java.base/sun.security.provider.certpath.PKIXMasterCertPathValidator.validate(PKIXMasterCertPathValidator.java:135)
        at java.base/sun.security.provider.certpath.PKIXCertPathValidator.validate(PKIXCertPathValidator.java:224)
        at java.base/sun.security.provider.certpath.PKIXCertPathValidator.validate(PKIXCertPathValidator.java:144)
        at java.base/sun.security.provider.certpath.PKIXCertPathValidator.engineValidate(PKIXCertPathValidator.java:83)       
        at java.base/java.security.cert.CertPathValidator.validate(CertPathValidator.java:309)
        at com.yubico.fido.metadata.FidoMetadataDownloader.verifyBlob(FidoMetadataDownloader.java:1097)
        at com.yubico.fido.metadata.FidoMetadataDownloader.parseAndVerifyBlob(FidoMetadataDownloader.java:1018)
        at com.yubico.fido.metadata.FidoMetadataDownloader.refreshBlobInternal(FidoMetadataDownloader.java:807)
        at com.yubico.fido.metadata.FidoMetadataDownloader.loadCachedBlob(FidoMetadataDownloader.java:712)
        at toy.Test.main(Test.java:29)
Caused by: java.security.cert.CertPathValidatorException: Could not determine revocation status
        at java.base/sun.security.provider.certpath.RevocationChecker.buildToNewKey(RevocationChecker.java:1152)
        at java.base/sun.security.provider.certpath.RevocationChecker.verifyWithSeparateSigningKey(RevocationChecker.java:972)
        at java.base/sun.security.provider.certpath.RevocationChecker.checkCRLs(RevocationChecker.java:611)
        at java.base/sun.security.provider.certpath.RevocationChecker.checkCRLs(RevocationChecker.java:471)
        at java.base/sun.security.provider.certpath.RevocationChecker.check(RevocationChecker.java:373)
        at java.base/sun.security.provider.certpath.RevocationChecker.check(RevocationChecker.java:343)
        at java.base/sun.security.provider.certpath.PKIXMasterCertPathValidator.validate(PKIXMasterCertPathValidator.java:125)
        ... 9 more
```

Is there something incredibly obvious that I am missing? I am using `openjdk 17.0.3 2022-04-19`.

--- a/webauthn-server-attestation/README.adoc
+++ b/webauthn-server-attestation/README.adoc
@@ -120,7 +120,7 @@ Breaking changes to these will NOT be reflected in version numbers.
 
 == Getting started
 
-Using this module consists of 4 major steps:
+Using this module consists of 5 major steps:
 
  1. Create a
     link:https://developers.yubico.com/java-webauthn-server/JavaDoc/webauthn-server-attestation/2.4.0/com/yubico/fido/metadata/FidoMetadataDownloader.html[`FidoMetadataDownloader`]
@@ -218,13 +218,16 @@ RegistrationResult result = rp.finishRegistration(/* ... */);
 Set<MetadataBLOBPayloadEntry> metadata = mds.findEntries(result);
 ----------
 
-By default,
-link:https://developers.yubico.com/java-webauthn-server/JavaDoc/webauthn-server-attestation/2.4.0/com/yubico/fido/metadata/FidoMetadataDownloader.html[`FidoMetadataDownloader`]
-will probably use the SUN provider for the `PKIX` certificate path validation algorithm.
-This requires the `com.sun.security.enableCRLDP` system property set to `true` in order to verify the BLOB signature.
+ 5. If you use the SUN provider for the `PKIX` certificate path validation algorithm, which many deployments do by default:
+    set the `com.sun.security.enableCRLDP` system property to `true`.
+    This is required for the SUN `PKIX` provider to support the CRL Distribution Points extension,
+    which is needed in order to verify the BLOB signature.
++
 For example, this can be done on the JVM command line using a `-Dcom.sun.security.enableCRLDP=true` option.
-See the https://docs.oracle.com/javase/9/security/java-pki-programmers-guide.htm#JSSEC-GUID-EB250086-0AC1-4D60-AE2A-FC7461374746[Java PKI Programmers Guide]
+See the https://docs.oracle.com/javase/9/security/java-pki-programmers-guide.htm#GUID-EB250086-0AC1-4D60-AE2A-FC7461374746__SECTION-139-623E860E[Java PKI Programmers Guide]
 for details.
++
+This step may not be necessary if you use a different provider for the `PKIX` certificate path validation algorithm.
 
 
 == Selecting trusted authenticators

